### ISSUE TITLE
Parse Error: Unique global class

### ISSUE THREAD

**Issue by git2013vb** [godotengine/godot#35442]
**Godot version:**
3.2rc2
**OS/device including version:**
Debian 10
**Issue description:**
I moved goap_action_list.gd in res://Goap/Action_List.gd but I receive this error:
```
SCRIPT ERROR: GDScript::reload: Parse Error: Unique global class "Action_List" already exists at path: res://Goap/Action_List.gd
   At: res://goap_action_list.gd:2.
```
res://goap_action_list.gd don't exist 

**Steps to reproduce:**
na
**Minimal reproduction project:**
na

**Comment by KoBeWi** [comment]
This needs more information and a reproduction project.

**Comment by Calinou** [comment]
@git2013vb Please upload a minimal reproduction project to make this easier to troubleshoot.

**Comment by EdmarDev** [comment]
I had this issue as well and can give more information:
When moving a script that has a `class_name` defined to a different folder, godot claims that the name already exists (from the previous location) and shows the unique global class error message. Restarting the editor seems to fix it.

**Comment by m312z** [comment]
I have the same issue, and restarting the editor does not fix it.
The file does not exist at the previous location, but opens in the editor with the error. Copying and pasting the script path shows the old path. Editing and saving the script then creates a duplicate at the old path.
I cannot find where the old path has been cached, and I can no longer work on this project.

**Comment by HiddenGoblin** [comment]
I have the same issue and can confirm @m312z.

**Comment by Calinou** [comment]
@m312z Try removing the editor user data directories (see [Data paths](https://docs.godotengine.org/en/latest/tutorials/io/data_paths.html)). Alternatively, you can move your project folder so the per-project cache isn't picked up.

**Comment by akien-mga** [comment]
Fixed by #40781.

**Comment by slakibk** [comment]
I solved this problem like this: I edited the old paths in the .tscn files.
